subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/LargeDependencyGraphPerformanceTest.groovy

```diff
@@ -94,26 +86,46 @@ class LargeDependencyGraphPerformanceTest extends AbstractCrossVersionPerformanc
         stopServer()
     }
 
-    private static BuildExperimentListener createArgsTweaker(String baseline) {
-        new BuildExperimentListener() {
-            @Override
-            void beforeExperiment(BuildExperimentSpec experimentSpec, File projectDir) {
-                GradleInvocationSpec invocation = experimentSpec.invocation as GradleInvocationSpec
-                if (invocation.gradleDistribution.version.version != baseline) {
-                    invocation.args << '-Dorg.gradle.advancedpomsupport=true'
-                }
-            }
+    def "resolve large dependency graph with advanced pom support"() {
+        runner.testProject = TEST_PROJECT_NAME
+        startServer()
+
+        given:
+        def baseline = "4.6-20180125002142+0000"
+        runner.tasksToRun = ['resolveDependencies']
+        runner.gradleOpts = ["-Xms256m", "-Xmx256m"]
+        runner.targetVersions = [baseline]
+        runner.args = ['-PuseHttp', "-PhttpPort=${serverPort}", '-PnoExcludes', '-Porg.gradle.advancedpomsupport=true']
+
+        when:
+        def result = runner.run()
+
+        then:
+        result.assertCurrentVersionHasNotRegressed()
+
+        cleanup:
+        stopServer()
+    }
+
+    def "resolve large dependency graph with advanced pom support (parallel)"() {
```

Comment:
I don't think we need this variant of the test. Our long term goal is to make parallelization something that 'just happens' and to deprecate the `--parallel` flag.